src/main/java/practice/StreamPractice.java

```diff
@@ -35,7 +53,10 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      * Example: select men who can be recruited to army (from 18 to 27 years old inclusively).
      */
     public List<People> selectMenByAge(List<People> peopleList, int fromAge, int toAge) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(people -> people.getSex() == People.Sex.MAN
+                        && people.getAge() >= fromAge && people.getAge() <= toAge)
```

Comment:
```suggestion
                        && people.getAge() >= fromAge
                        && people.getAge() <= toAge)
```
Так более читабельно)))